operator/hack/common.sh

```diff
@@ -65,6 +65,15 @@ function retry() {
   return 1
 }
 
+function check_version_tag() {
+  local -r version_tag="$1"
+
+    if [[ $version_tag == *-dirty ]]; then
+      log "Cannot install from *-dirty image tag. Dirty tag images are not supposed to be pushed to the registry"
+      return 1
+    fi
```

Comment:
Please use consistent indentation
```suggestion
  if [[ $version_tag == *-dirty ]]; then
    log "Cannot install from *-dirty image tag. Dirty tag images are not supposed to be pushed to the registry."
    return 1
  fi
```